cloud/google/config.sh

```diff
@@ -28,26 +31,31 @@ for i in $(seq ${LOG_NUM_REPLICAS}); do
 done
 
 
-export PROMETHEUS_NUM_REPLICAS=1
-export PROMETHEUS_DISK_SIZE=50GB
-export PROMETHEUS_BASE_NAME="${CLUSTER}-prometheus"
-export PROMETHEUS_MACHINE_TYPE=n1-standard-1
-declare -a PROMETHEUS_MACHINES LOG_DISKS
-export PROMETHEUS_MACHINES LOG_DISKS
-for i in $(seq ${PROMETHEUS_NUM_REPLICAS}); do
-  PROMETHEUS_MACHINES[$i]="${CLUSTER}-prometheus-$i"
-  PROMETHEUS_DISKS[$i]="${CLUSTER}-prometheus-disk-$i"
-done
+if [ "${MONITORING}" == "prometheus" ]; then
```

Comment:
Not really - we just don't run stuff. Arguably you might want to configure the [new] `..._base_url` flag, but I'd say we'll add that when/if we need it.